src/components/products/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { Link } from "react-router-dom";
+
+import FullStar from '../../stars/FullStart'
+import HalfStar from '../../stars/HalfStar'
+import EmptyStar from '../../stars/EmptyStar'
+
+import './ProductCard.css'
+
+
+const ProductCard = ({imgUrl, price, name, average_rating, isOnSale, item_id}) => {
+
+    const stars = []
```

Comment:
Nice job calculating the stars programmatically here! However, you should create a component for them to encapsulate the logic. You will also need stars on the item page.

Reply:
Ok thanks. Since I will use stars again I might repeat myself